examples/navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import 'rxjs';
```

Comment:
Comment added in https://github.com/redux-observable/redux-observable/pull/89/commits/e885d0ad45f73b9c9405a9044a029a179a3f11cb